data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
```

Comment:
I could imagine there might be cases where a work has several commonly used titles that we might want to include for searchability purposes (even if they are never directly attested in the mss and they aren't the uniform titles). Do we want a field where we define ‘aka’/‘alternate names’? Would this be in addition to a field for ‘attested names’, which would come from textual artifacts?